src/redux/missions/mission.js

```diff
@@ -1,9 +1,21 @@
 const GET_MISSIONS = 'bookStore/books/GET_MISSIONS';
+const BASE_URL = 'https://api.spacexdata.com/v3/missions';
 const initialState = [];
 
+export const getMission = async (dispatch) => {
+  const request = await fetch(BASE_URL);
+  if (request.status === 200) {
+    const payload = await request.json();
+    dispatch({
+      type: GET_MISSIONS,
+      payload,
+    });
+  }
+};
+
 const missionReducer = (state = initialState, action) => {
   switch (action.type) {
-    case GET_MISSIONS: return state;
+    case GET_MISSIONS: return action.payload;
```

Comment:
In this section, I am just aware that this action deletes the data that I have stored from rockets.
I was thinking you can change the return statement to something like this:

``
return {
  ...state,
  missions: [...action.payload],
}
``

in my case I initialize the store as an object with a rockets array inside 
``
  store = {
    rockets: [ ],
  }
``

Reply:
I am not really sure if this change is necessary. I just want to let you know, otherwise, everything is ok for me.